src/test/java/spoon/test/eval/EvalTest.java

```diff
@@ -311,7 +315,35 @@ public void testconvertElementToRuntimeObject() {
 
 	}
 
-	private CtBinaryOperator<?> createBinaryOperatorOnLiterals(Factory factory, Object leftLiteral, Object rightLiteral, BinaryOperatorKind opKind) {
-		return factory.createBinaryOperator(factory.createLiteral(leftLiteral), factory.createLiteral(rightLiteral), opKind);
+	private CtBinaryOperator<?> createBinaryOperatorOnLiterals(Factory factory, Object leftLiteral,
+			Object rightLiteral, BinaryOperatorKind opKind) {
+		return factory.createBinaryOperator(factory.createLiteral(leftLiteral),
+				factory.createLiteral(rightLiteral), opKind);
+	}
+	
+	@ParameterizedTest
+	@CsvSource(
+	delimiter = '|',
+	useHeadersInDisplayName = true,
+	value = {
+	" Literal  | Expected ",	
+	"-1.234567 | -1.234567",
+	"-2.345F   | -2.345F   ", 
+	"-3        | -3 		  ",
+	"-4L 		   | -4L       "})
```

Comment:
This looks odd. Maybe this instead?
```suggestion
	@CsvSource(
		delimiter = '|',
		useHeadersInDisplayName = true,
		value = {
			" Literal  | Expected ",	
			"-1.234567 | -1.234567",
			"-2.345F   | -2.345F   ", 
			"-3        | -3 		  ",
			"-4L 		   | -4L       "
		}
	)
```